wagtailimporter/serializer.py

```diff
@@ -7,11 +7,11 @@
 from pathlib import PurePosixPath
 
 import yaml
-from unidecode import unidecode
```

Comment:
Unidecode is removed from Wagtail, so removed dependency here too and pulled in new code for Wagtail behaviour below